### DISCUSSION TITLE
PyMYSQL connection issue with dble.

### DISCUSSION THREAD

**Discussion by AYO-Al** [PyMySQL/PyMySQL#1149] [Q&A]
**Describe the bug**
Why does Django, using PyMySQL as the MYSQL engine, fail to perform insert and update operations on the database after connecting to dble, while delete operations work?

**To Reproduce**
Complete steps to reproduce the behavior:
1. Install PyMySQL as the Django database engine.
2. Configure the management port opened by dble with the username and password both set to 'root', and the port set to 8066.
3. After starting the Django project, it was observed that only deletion of data is possible, and operations for addition or modification are not working.
4. Upon reviewing the logs, it was found that the requests are normal, but only the delete operation is taking effect.
5. Directing to the database port 3306 restored normal data operations.

**Environment**
 - OS: CentOS7
 - Server and version: MySQL 5.7
 - PyMySQL version: 1.0.2

**Comment by methane** [comment]
I can not help you without concrete reproducer.
Ask help to dble community.
